charts/jenkins/README.md

```diff
@@ -703,4 +706,19 @@ Upgrade an existing release from `stable/jenkins` to `jenkins/jenkins` seamlessl
 
 Chart release versions follow [SemVer](../../CONTRIBUTING.md#versioning), where a MAJOR version change (example `1.0.0` -> `2.0.0`) indicates an incompatible breaking change needing manual actions.
 
-See [UPGRADING.md](./UPGRADING.md) for a list of breaking changes
+See [UPGRADING.md](./UPGRADING.md) for a list of breaking change
+
+<!-- textlint-disable terminology -->
+## Maintainers
+
+| Name | Email | Url |
```

Comment:
url is empty and never used which looks weird.

I'm not sure if we need maintainers in here really

Reply:
See comment above